Dockerfile

```diff
@@ -38,7 +38,7 @@ RUN apt-get -y autoremove && apt-get clean && apt-get autoclean && \
 RUN mkdir -p /var/talis-php
 COPY . /var/talis-php
 
-RUN echo "export PERSONA_TEST_HOST='https://staging-users.talis.com'" >> /etc/profile.d/test.sh \
+RUN echo "export PERSONA_TEST_HOST='http://persona.talis.local'" >> /etc/profile.d/test.sh \
```

Comment:
The instructions previously required you to have oauth client and secret for an `su` scoped client in staging.

This PR changes this to run against persona.talis.local and details how to create the client.